test/coreneuron/test_direct.py

```diff
@@ -40,9 +40,8 @@ def test_direct_memory_transfer():
     pc.psolve(h.tstop)
     tran = [h.t, h.soma(.5).v, h.soma(.5).hh.m]
 
-    assert(v.eq(vstd))
```

Comment:
@olupton @alexsavulescu : this check is very helpful to detect bugs which may not be captured by tolerance `1e-10`. So I was thinking of more parametrising this e.g. if it's debug build (O0 or with GCC) then do exact checks otherwise use tolerance.

Reply:
Actually this check is new, see https://github.com/neuronsimulator/nrn/pull/990#discussion_r595913767 
We can open a ticket for that if you deem it useful

Reply:
Interesting! I didn't realise that was added in recent times.

I think having flexibility to check identical results would be helpful. Otherwise you don't realise when some discrepancy is added and we will notice it only when something will cross the tolerance.

Reply:
In that case we need to see why we had different values on Jureca. For sure this new check was avoided for some reason in the past?